ql/pom.xml

```diff
@@ -820,6 +820,12 @@
       <version>${guava.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
+      <artifactId>fastutil</artifactId>
+      <version>6.5.6</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
https://github.com/apache/hive/blob/4b670877c280b37c5776046f66d766079489b2a8/ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestWorker.java#L20 need this dependency:
![image](https://user-images.githubusercontent.com/5399861/100756200-f6910480-3427-11eb-9919-af782b870b9e.png)